File: Back End Spring/src/main/java/org/elis/movieexplorer/dto/chat/response/ResponseInfoChatDTO.java
package org.elis.movieexplorer.dto.chat.response;

import org.elis.movieexplorer.dto.message.response.ResponseMessaggioDTO;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ResponseInfoChatDTO {
	private Long id;
	
	private String oggetto;
	
	private String stato;
	
	private Boolean messaggiInSospeso;
	
	private String nome;
	
	private String cognome;
		
	private ResponseMessaggioDTO ultimoMessaggio;
}

File: Front End Angular/src/app/dto/chat/response/ResponseInfoChatDTO.ts
import {ResponseMessaggioDTO} from "../../message/response/ResponseMessaggioDTO";

export interface ResponseInfoChatDTO {
    id: number;
    oggetto: string;
    stato: string;
    messaggiInSospeso: boolean;
    nome: string;
    cognome: string;
    ultimoMessaggio: ResponseMessaggioDTO;
}